packages/issuer-sdk-js/README.md

```diff
@@ -9,13 +9,123 @@ Get [our NPM package](https://www.npmjs.com/package/@idos-network/idos-sdk) and
 pnpm add @idos-network/idos-issuer-sdk-js
 ```
 
+
+## Quickstart
```

Comment:
This is not a quickstart anymore, it's just the setup. Let's reflect that in the section name. Using gerund to keep consistent with (at least most of) the rest of the headers.

```suggestion
## Setting up
```